### DISCUSSION TITLE
lost the faikin in my network?

### DISCUSSION THREAD

**Discussion by sjaakie09** [revk/xESP32-Faikout#280] [General]
my second unit was running for two weeks now and today i could not find it anymore on the normal ip addres? 192.168.1.156 and i did a lan scan and dot see it anymore ? the info on the scan should be  http://daikin.local/  i give it that name? the unit reacts on when the airco is on heating mode then the led is bllinking red  so there is com between the faikin and the daikin but no wifi how can I get it back in factory settings? please assist thx jacques
![faikinv2](https://github.com/revk/ESP32-Faikin/assets/159342967/2dc8991f-a139-4888-80c2-8c1812c83b03)

 bought via amazone UK

**Comment by wurmtal868** [comment]
Eventually the Faikin module got a new IP address assigned. 
Did you check your Router / DHCP server for known devices?

**Comment by timfrazee** [comment]
Looks like you may have a typo. Your unit is named "Dakin" in the screenshot, so the URL should be http://dakin.local.

Otherwise, what wurmtal868 said - your router assigned it a new IP, so you'll have to look it up there.
